demonstrations/tutorial_mps.py

```diff
@@ -0,0 +1,698 @@
+r"""Introducing matrix product states for quantum practitioners
+===============================================================
+
+Matrix product states remain the workhorse for a broad range of modern classical quantum simulation techniques,
+still to this day. Their unique features (like offering a canonical form) make them an incredibly neat tool
+in terms of simplicity and algorithmic complexity.
+In this demo, we are going to cover all the essentials you need to know in order to handle matrix product states,
+and show how to use them to simulate quantum circuits.
+
+.. figure:: ../_static/demonstration_assets/how_to_simulate_quantum_circuits_with_tensor_networks/TN_MPS.gif
+    :align: center
+    :width: 90%
+    
+Introduction
+------------
+
+Matrix product states (MPS) are an efficient representation of quantum states in one spatial dimension.
+However, due to their unique features like offering a canonical form, they are employed in a variety of tasks beyond just 1D systems.
+
+The amount of entanglement the MPS can represent is user-controlled via a hyper-parameter, the so-called `bond dimension` :math:`\chi`.
+If we allow :math:`\chi` to be of :math:`\mathcal{O}(2^{\frac{n}{2}})` for a system of :math:`n` qubits, we can write `any` state as an `exact` MPS.
+To avoid exponentially large resources, however, one typically sets a finite bond dimension :math:`\chi` at the cost of introducing an approximation error.
+
+.. figure:: ../_static/demo_thumbnails/opengraph_demo_thumbnails/OGthumbnail_mps_simulation.png
+    :align: center
+    :width: 40%
```

Comment:
Make it 70% unless you really hate that size. We're trying to keep these intro visuals consistent in line thickness.
```suggestion
    :width: 70%
```